lib/manageiq/loggers/deprecator.rb

```diff
@@ -0,0 +1,7 @@
+module ManageIQ
+  module Loggers
```

Comment:
This being in a file called deprecator.rb but not providing the Deprecator constant may cause issues with zeitweirk.  Perhaps we don't need the extra file and can just have this method live in lib/manageiq/loggers.rb?